pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/mercury/GetMercuryEligibleSamplesRoute.java

```diff
@@ -16,14 +17,17 @@
 @Slf4j
 public class GetMercuryEligibleSamplesRoute extends RequestHandler {
     private MercurySampleDao mercurySampleDao;
+    private KitDaoImpl kitDaoImpl;
     public static String projectId;
     public static String topicId;
```

Comment:
I know this hasn't been changed in terms of this PR, but I see that those two fields are static ones and we initialize them in the constructor. Is that expected behavior?

Reply:
Yeah they are just values that are read from secret, that's why we don't have it at this point to initialize it